src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2728,6 +2730,10 @@ def aks_update(cmd, client, resource_group_name, name,
                 '--rotation-poll-interval can only be specified when azure-keyvault-secrets-provider is enabled')
         azure_keyvault_secrets_provider_addon_profile.config[CONST_ROTATION_POLL_INTERVAL] = rotation_poll_interval
 
+    if nodepool_labels is not None:
+        for agent_profile in instance.agent_pool_profiles:
+            agent_profile.node_labels = nodepool_labels
```

Comment:
This would update all the pools, is this expected? Do we need to remind users?

Reply:
Yes. If customer use `aks update --nodepool-labels` to update labels, it means labels on all node pools will be updated to the same value. if they wish to update different label on different node pool separately, they should use `aks nodepool update --labels`. We can make this clear in our doc when add these new commands.